build/esbuild-build.ts

```diff
@@ -33,3 +35,17 @@ esbuild
     console.error(err);
     process.exit(1);
   });
+
+function createEnvDefines(variableNames: string[]): Record<string, string> {
+  const defines: Record<string, string> = {};
+  dotenv.config();
+  for (const name of variableNames) {
+    const envVar = process.env[name];
+    if (envVar !== undefined) {
+      defines[`process.env.${name}`] = JSON.stringify(envVar);
```

Comment:
I think this might be overdone. You just call the variables from the global scope. 

Check my implementation of passing in the RPCs 

https://github.com/ubiquity/pay.ubq.fi/pull/167

Reply:
Just copied the code you made for another project I don't remember which one, will change it